docs/course_materials/exercises/01_Linux/check_homework_1.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+import check_helper
+
+exercise_commands = {
+    'pwd': 'printing current directory name',
+    'mkdir': 'creating directories',
+    'cd': 'changing between directories',
+    'ls': 'listing directory contents',
+    'tree': 'viewing the directory tree',
```

Comment:
I  think it's better to have a full dictionary in the helper of entire commands and their description to not repeat ourselves, and in here you can just have an array with the names of the commands that are relevant to the specific exercise.